packages/azure/data_stream/firewall_logs/elasticsearch/ingest_pipeline/default.yml

```
@@ -41,7 +41,145 @@ processors:
       field: json.resourceId
       target_field: azure.resource_id
       ignore_missing: true
-
+  - rename:
+      field: json.properties.DestinationIp
+      target_field: destination.ip
+      ignore_missing: true
+  - rename:
+      field: json.properties.DestinationPort
+      target_field: destination.port
+      ignore_missing: true
+  - rename:
+      field: json.properties.Policy
+      target_field: azure.firewall.policy
+      ignore_missing: true
+  - rename:
+      field: json.properties.Protocol
+      target_field: azure.firewall.protocol
+      ignore_missing: true
+  - rename:
+      field: json.properties.Rule
+      target_field: rule.name
+      ignore_missing: true
+  - rename:
+      field: json.properties.RuleCollection
+      target_field: rule.ruleset
+      ignore_missing: true
+  - rename:
+      field: json.properties.RuleCollectionGroup
+      target_field: azure.firewall.rule_collection_group
+      ignore_missing: true
+  - rename:
+      field: json.properties.SourceIp
+      target_field: source.ip
+      ignore_missing: true
+  - rename:
+      field: json.properties.SourcePort
+      target_field: source.port
+      ignore_missing: true
+  - rename:
+      field: json.properties.TranslatedIp
+      target_field: destination.ip
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      field: json.properties.TranslatedPort
+      target_field: destination.port
+      ignore_missing: true
+      ignore_failure: true
+      if: ctx?.azure?.firewall?.properties?.DestinationPort == null
+  - rename:
+      field: json.properties.Action
+      target_field: azure.firewall.action
+      ignore_missing: true
+  - rename:
+      field: json.properties.ActionReason
+      target_field: azure.firewall.action_reason
+      ignore_missing: true
+  - rename:
+      field: json.properties.Fqdn
+      target_field: destination.domain
+      ignore_missing: true
+  - rename:
+      field: json.properties.IsExplicitProxyRequest
+      target_field: azure.firewall.is_explicit_proxy_request
+      ignore_missing: true
+  - rename:
+      field: json.properties.IsTlsInspected
+      target_field: azure.firewall.is_tls_inspected
+      ignore_missing: true
+  - uri_parts:
+      field: json.properties.TargetUrl
+      if: ctx.azure?.firewall?.properties?.TargetUrl != null
```

```suggestion
      if: ctx.json?.properties?.TargetUrl != null
      tag: uri_parts_targeturl
```

Add `tag` since you have tag reference in `error.message`